packages/target-starknet-test/package.json

```diff
@@ -26,6 +26,6 @@
   },
   "dependencies": {
     "@types/elliptic": "^6.4.14",
-    "starknet": "^3.9.0"
```

Comment:
any specific reason for this change?

Reply:
Did this in [2085673](https://github.com/dethcrypto/TypeChain/pull/803/commits/2085673e067a6b800aad221d83153c733e232af6) temporarily to try to fix CI along with [6ce08b5](https://github.com/dethcrypto/TypeChain/pull/803/commits/6ce08b5b77635a57640d67af06cbc5d8c83378b7) and [eacbe6d](https://github.com/dethcrypto/TypeChain/pull/803/commits/eacbe6d03d9a98dcec8d38b1384d7fc6125b907e). Reverting them.